### ISSUE TITLE
Lazy Loading Angular Cards

### ISSUE THREAD

**Issue by alabgold** [aFarkas/lazysizes#289]
Hello, my site is using angular cards to hold images. They are angular material outlined in this demo: https://material.angularjs.org/latest/demo/card 
I was wondering if lazy sizes could be adapted to work on these cards?

**Comment by aFarkas** [comment]
Yes, you need to set `data-src` instead of `src`/`ng-src`. Also note, that in case you are dynamically changing the `data-src` attribute you also need to re-add the `lazyload` class. You can also add the attrchange plugin to automate this.

**Comment by alabgold** [comment]
@aFarkas should class="lazyload" go in the card class, or with the image in the card?

**Comment by aFarkas** [comment]
On the `img` element. If you provide a fiddle with the cards, I can add lazysizes for you.

**Comment by alabgold** [comment]
https://jsfiddle.net/alabgold/0tmcosmn/

**Comment by aFarkas** [comment]
Actually I meant a **working**  angular cards fiddle. From markup:

You will need something like this:

```html
<div style="border: 1px solid black;">
     <img class="card-img lazyload" data-src="{{slide.url}}">
</div>
```

It might be also wise to start lazysizes after angualr has initialized.

You can do this:

```js
window.lazySizesConfig = window.lazySizesConfig || {};
window.lazySizesConfig.init = false;

//call lazysizes later:
lazySizes.init();
```
